src/workflows-app/components/OutputsTable.test.js

```diff
@@ -2,7 +2,7 @@ import { render, screen, within } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { h } from 'react-hyperscript-helpers';
 import OutputsTable from 'src/workflows-app/components/OutputsTable';
-import { runSetOutputDef, runSetOutputDefWithDefaults } from 'src/workflows-app/utils/mock-responses';
+import { runSetOutputDef, runSetOutputDefEmpty, runSetOutputDefWithDefaults } from 'src/workflows-app/utils/mock-responses';
```

Comment:
this isn't directly related to your PR, but because you're adding something to this file `mock-responses` I feel like I should bring this up:

I think this file's name and location isn't quite right... these are more like "sample configurations" than "mock responses". I created a file at `src/libs/ajax/workflows-app/cbas-sample-configurations.js` for use in the CBAS contract tests in that same directory, but now these files are super similar.

It's not really in scope of this ticket, so I'm approving, but at some point I think we should consolidate these two files into one -- my suggestion would be `src/testing/workflows-app-sample-configurations.js`